sections/introduction.include

```diff
@@ -392,10 +392,10 @@
   anonymize requests so that requests from a single user at one node on the Internet appear to come
   from many disparate parts of the network.
 
-  However, the IP address used for a user's requests is not the only mechanism by which a user's
-  requests could be related to each other. Cookies, for example, are designed specifically to enable
-  this, and are the basis of most of the Web's session features that enable you to log into a site
-  with which you have an account.
+  However, the IP address used for a user's requests is not the only mechanism by which different
+  requests of a single user could be related to each other. Cookies, for example, are designed
+  specifically to enable this, and are the basis of most of the Web's session features that enable
+  you to log into a site with which you have an account.
```

Comment:
Final sentence is getting long. Consider two sentences...

specifically to enable this. They are the basis [...] features **enabling you** to log [...].